controllers/DefaultController.php

```diff
@@ -50,8 +51,22 @@ public function actionRpc()
 
         switch ($options['method']) {
             case 'yii':
-                list ($status, $output) = $this->runConsole(implode(' ', $options['params']));
+	            $cmd = Yii::getAlias($this->module->yiiScript) . ' '
```

Comment:
Instead of moving building of the command here I'd introduce `runYiiCommand` and `runComposerCommand`.